freshmaker/errata.py

```diff
@@ -78,6 +84,9 @@ def from_advisory_id(cls, errata, errata_id):
         else:
             cve_list = []
 
+        # security_impact in errata is capitalized string
+        security_impact = erratum_data["security_impact"].lower()
```

Comment:
Is making this lowercase for backwards compatibility with SFM2?

Reply:
Yes, comment is updated.